virttest/utils_disk.py

```diff
@@ -601,6 +603,164 @@ def create_filesystem(session, partition_name, fstype, ostype, timeout=360):
         create_filesyetem_linux(session, partition_name, fstype, timeout)
 
 
+def get_mpoint_fstype_linux(session, partition):
+    """
+    Get a partition's file system type and mountpoint.
+
+    :param session: session object to guest.
+    :param partition: disk partition, like /dev/sdb1.
+    :return: Tuple (mountpoint, fstype)
+    """
+    mount_list = session.cmd_output('cat /proc/mounts')
+    mount_info = re.search(r'%s\s(.+?)\s(.+?)\s' % partition, mount_list)
+    if mount_info:
+        return mount_info.groups()
+    raise exceptions.TestError('partition is not mounted, mounted'
+                               'list as:\n%s' % mount_list)
+
+
+def get_partition_attrs_linux(session, partition):
+    """
+    Get partition attributes in linux guest.
+
+    :param session: session object to guest.
+    :param partition: like /dev/sdb1.
+    :return: dict like {'start': '512B', 'end': '16106127359B',
+                       'size': '16106126848B', 'type': 'primary'}
+    """
+    pattern = r'(/dev/.*)p(\d+)' if "nvme" in partition else r'(/dev/.*)(\d+)'
+    dev_name, part_num = re.match(pattern, partition).groups()
+    parted_cmd = 'parted -s %s unit B print' % dev_name
+    pattern = re.compile(r'%s\s+(?P<start>\d+\w+)\s+(?P<end>\d+\w+)\s+'
+                         r'(?P<size>\d+\w+)\s+(?P<type>\w+)' % part_num)
+    return pattern.search(session.cmd(parted_cmd), re.M).groupdict()
+
+
+def resize_filesystem_linux(session, partition, size):
+    """
+    Resize file system in linux guest.
+    For ext2, ext3, ext4 filesystem, support enlarge and shrink.
+    For xfs filesystem, only support enlarge, not support shrink.
```

Comment:
As we discussed before, not support gpt for linux. Suggest to comment it. In order to prevent  calling incorrectly

Reply:
that's ok, will update~

Reply:
Updated, thanks~